### DISCUSSION TITLE
Why is MobileNetV2's robustness  better than ResNet50 ?

### DISCUSSION THREAD

**Discussion by Senhui-Qiu** [STVIR/pysot#200] [General]
@lb1100 
From PYSOT model zoo, it seems that MobileNetV2's robustness is better than ResNet50 in VOT19, so I have 3 questions to consult.
1、Why is MobileNetV2's robustness  better than ResNet50 ?
2、How to obtain or  train（method） pretrained backbones model of MobileNetV2?
3、Are you planning to upload the pretrained backbones model and configuration files of MobileNetV2?

thanks

**Comment by ZhiyuanChen** [comment]
Even though, the robustness has increased 10%,
which is the key vulnerability of our tracker. It means that
our tracker still has room for improvement. After aggregating
all three layers, both accuracy and robustness steadily
improve, with gains between 3:1% and 1:3% for VOT and
OTB.
